libexec/rtld-elf/Makefile

```diff
@@ -29,11 +29,19 @@ MK_UBSAN=	no
 
 .include <bsd.compat.pre.mk>
 
+.if (${MACHINE_ABI:Mpurecap} && ${MACHINE_CPUARCH} == "aarch64") || defined(RTLD_SANDBOX_ALWAYS)
+RTLD_SANDBOX=
+.endif
+
 .ifdef RTLD_SANDBOX
 CFLAGS+=-DRTLD_SANDBOX
 .endif
 
-.if !defined(NEED_COMPAT) && !defined(RTLD_SANDBOX)
```

Comment:
@jrtc27 I misunderstood the purpose of those `ifdef RTLD_SANDBOX`s in the Makefiles of libc/libc_c18n. But what about this line? I suppose I should change it so that `CONFS` is only defined once among all variants of RTLD?

Reply:
I.e., I should change it to ` && !defined(RTLD_SANDBOX_ALWAYS)`, where `RTLD_SANDBOX_ALWAYS` is a new variable that is only defined for `ld-elf[64cb]-c18n`.

Reply:
That would work; just needs to be something defined by the -c18n rtlds but not rtld-elf itself.